### DISCUSSION TITLE
Update/Change ShortURL (Custom slug)

### DISCUSSION THREAD

**Discussion by TritonB7** [shlinkio/discussions#107] [Help wanted] [answered]
After creating a short url, is it possible to modify the short url (Custom slug) in the future? For example if I created the following short url example.com/myCustomSlug100 , would I be able to change this to example.com/myCustomSlug200 at a later date?

**Comment by acelaya** [comment, marked as answer]
No, that's not possible. In order to do that, just create a new short URL.
